app/utils/constants.js

```diff
@@ -38,8 +38,13 @@ export const placeholderNietBeschikbaar = {
 };
 
 export const MANDATARIS_PREDICATE = {
-  persoon: 'http://data.vlaanderen.be/ns/mandaat#isBestuurlijkeAliasVan',
-  mandaat: 'http://www.w3.org/ns/org#holds',
+  persoon: new Namespace('http://data.vlaanderen.be/ns/mandaat#')(
```

Comment:
why change this? i don't see the value in first wrapping it to then unwrap it again in the isPredicateInObserverChange.
I'm ok with it though, but i wouldn't have done this. If we can use simple strings we should use simple strings imo